journey/ios/3.7.0/getting_started.md

```diff
@@ -181,19 +214,22 @@ The table below explains the different parameters that you are able to set befor
 - Example
 
 ```swift
+JourneySdk.shared.enviroment = .prod
 JourneySdk.shared.formJourney = false
 JourneySdk.shared.advancedSearchMode = false
 JourneySdk.shared.maxHistory = 12
 JourneySdk.shared.multiNetwork = true
 JourneySdk.shared.isEarlierLaterFeatureEnabled = true
 JourneySdk.shared.modeForm = [ModeButtonModel(title: "Metro",
```

Comment:
options for journey sdk should be specified with the configuration builder when configuration object is instantiated.